test/helpers/test-helper.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * Imports.
+ */
+const PhoenixClient = require('../../lib/phoenix-client');
```

Comment:
As I understand, this is a common practice in NodeJs.
Though, I don't like it either. @DFurnes is there a better way to do it?